docs/Using-Fleet/configuration-files/README.md

```diff
@@ -264,15 +270,928 @@ spec:
+- Default value: `false`
+- Config file format:
+  ```
+  sso_settings:
+    enable_jit_provisioning: true
+  ```
+
+#### sso_settings.entity_id
+
+The required entity ID is a Uniform Resource Identifier (URI) that you use to identify Fleet when configuring the identity provider. It must exactly match the Entity ID field used in identity provider configuration.
+
+- Required setting if SSO is enabled, must have at least 5 characters (string).
+- Default value: none.
+- Config file format:
+  ```
+  sso_settings:
+    entity_id: "https://example.com"
+  ```
+
+#### sso_settings.idp_image_url
+
+An optional link to an image such as a logo for the identity provider.
+
+- Optional setting (string).
+- Default value: none.
+- Config file format:
+  ```
+  sso_settings:
+    idp_image_url: "https://example.com/logo"
+  ```
+
+#### sso_settings.idp_name
+
+A required human-friendly name for the identity provider that will provide single sign-on authentication.
+
+- Required setting if SSO is enabled (string).
+- Default value: none.
+- Config file format:
+  ```
+  sso_settings:
+    idp_name: "SimpleSAML"
+  ```
+
+#### sso_settings.issuer_uri
+
+The issuer URI supplied by the identity provider. TODO(mna): is this used? The field does not appear used anywhere. We do request it on the frontend.
+
+- Optional setting (string).
+- Default value: none.
+- Config file format:
+  ```
+  sso_settings:
+    issuer_uri: "https://example.com/saml2/sso-service"
+  ```
+
+#### sso_settings.metadata
+
+Metadata (in XML format) provided by the identity provider.
+
+- Optional setting, either `metadata` or `metadata_url` must be set if SSO is enabled, but not both (string).
+- Default value: none.
+- Config file format:
+  ```
   sso_settings:
-    enable_sso: false
-    entity_id: 1234567890
-    idp_image_url: https://idp.example.org/logo.png
-    idp_name: IDP Vendor 1
-    issuer_uri: https://idp.example.org/SAML2/SSO/POST
     metadata: "<md:EntityDescriptor entityID="https://idp.example.org/SAML2"> ... /md:EntityDescriptor>"
+  ```
+
+#### sso_settings.metadata_url
+
+A URL that references the identity provider metadata.
+
+- Optional setting, either `metadata` or `metadata_url` must be set if SSO is enabled, but not both (string).
+- Default value: none.
+- Config file format:
+  ```
+  sso_settings:
     metadata_url: https://idp.example.org/idp-meta.xml
-```
+  ```
+
+### Vulnerability Settings
+
+#### vulnerability_settings.databases_path
+
+Path to a directory on the local filesystem (accessible to the Fleet server) where the various vulnerability databases will be stored.
+
+- Optional setting, must be set to enable vulnerability detection (string).
+- Default value: none.
+- Config file format:
+  ```
+  vulnerability_settings:
+    databases_path: "/path/to/dir"
+  ```
+
+### Webhook Settings
+
+<!-- This section used to be named Webhooks, this ensures links with the #webhooks hash still work -->
+<span id="webhooks" name="webhooks"></span>
```

Comment:
```suggestion
```

I think we can remove this backlink.

I searched the Fleet repo and there are no links to the old section.